.github/workflows/aws-lambda-microvm-release.yml

```
@@ -21,124 +24,207 @@ concurrency:
-        id: publish
+          role-session-name: hackerai-microvm-${{ github.run_id }}-${{ strategy.job-index }}
+      - name: Publish exact regional image version
         env:
-          AWS_LAMBDA_MICROVM_OUTPUT_FILE: ${{ runner.temp }}/microvm-release.env
+          AWS_LAMBDA_MICROVM_OUTPUT_FILE: ${{ runner.temp }}/${{ matrix.region }}.env
+        run: node scripts/deploy-aws-microvm.mjs
+      - name: Launch and terminate exact regional image version
+        env:
+          AWS_LAMBDA_MICROVM_OUTPUT_FILE: ${{ runner.temp }}/${{ matrix.region }}.env
         shell: bash
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2163:warning:5:14: This does not export 'name'. Remove $/${} for that, or use ${var?} to quiet

(shellcheck)